lib/rails_response_dumper/colorize.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+COLORS = {
+  red: 31,
+  cyan: 36,
+  green: 32
```

Comment:
Is this in any particular order? I think it should either be alphabetical or numerical order.